gooey/tests/test_time_remaining.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+from argparse import ArgumentParser
+from itertools import *
+
+from tests.harness import instrumentGooey
+
+class TestFooterTimeRemaining(unittest.TestCase):
+
+    def make_parser(self):
+        parser = ArgumentParser(description='description')
+        return parser
+
+
+    # TODO: This test needs to have a progress loop to show the time remaining
```

Comment:
The `instrumentGooey` thing'll spit out the instantiated Gooey object, this will be built from whatever parser is provided, so you can use varying parser setups to test Gooey does the Right Thing. 

So, for example, if you wanted to test the presentational stuff is working as expected (and this is usually where a lot of Gooey regressions occur), you could put Gooey into the console mode (`gooeyApp.showConsole()`) and then assert that your timing info is visible/hidden. 

Similarly, you can test that your updates work by dispatching an event and then checking the results.

Reply:
Tests are now added and passing 👍